VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -298,7 +298,7 @@ public void onResult(DownloadEntry.WatchedState result) {
                     if(ws == null || ws == DownloadEntry.WatchedState.UNWATCHED) {
                         viewHolder.rowType.setIconColor(context.getResources().getColor(R.color.edx_brand_primary_base));
                     } else if(ws == DownloadEntry.WatchedState.PARTIALLY_WATCHED) {
-                        viewHolder.rowType.setIconColor(context.getResources().getColor(R.color.edx_grayscale_neutral_base));
+                        viewHolder.rowType.setIconColor(context.getResources().getColor(R.color.edx_brand_primary_base));
```

Comment:
>Since this block is now identical to the one above it, they could be combined

I think it looks better as it is, combining the `if` and `else-if`block will affect its readability instead of improving it.

 >I suggest making this a switch statement, and combining identical states.

That will require us to combine `if-else` with `switch` statement, because `ws` can be `null` too.